guides/source/active_job_basics.md

```diff
@@ -34,7 +34,7 @@ Delayed Job and Resque. Picking your queuing backend becomes more of an operatio
 concern, then. And you'll be able to switch between them without having to rewrite
 your jobs.
 
-NOTE: Rails by default comes with an asynchronous queuing implementation that
+NOTE: Rails by default come with an asynchronous queuing implementation that
```

Comment:
This is not an improvement as `Rails` is singular in this context, and most of the documentation.